src/buckets/components/createBucketForm/MeasurementSchemaUtils.ts

```diff
@@ -0,0 +1,75 @@
+import {trim} from 'lodash'
+
+const typeStrings = ['timestamp', 'tag', 'field']
+const dataTypeStrings = ['integer', 'float', 'boolean', 'string', 'unsigned']
+
+/**
+ * manually checking that the literal string union types are correct
+ *
+ * our types are generated, and changing it so there is an exported array that is *then* used
+ * for literal union type creation is out of scope.
+ * https://newbedev.com/checking-validity-of-string-literal-union-type-at-runtime
```

Comment:
we have [prior examples](https://github.com/influxdata/ui/blob/master/src/types/buckets.ts#L5) of extending generated types if we need to. Not sure if that's what this comment is saying...

Reply:
it's saying that in order to reuse the string arrays, the literal union types need to be generated differently.  so I am just retyping the string arrays

Reply:
cool, I figured I misunderstood the comment